bears/hypertext/HTMLLintBear.py

```diff
@@ -28,9 +28,101 @@ class HTMLLintBear:
 
     @staticmethod
     def create_arguments(filename, file, config_file,
+                         enable_doctype: bool=True,
+                         enable_entities: bool=True,
+                         enable_trailing_whitespace: bool=True,
+                         enable_tabs: bool=True,
+                         enable_charset: bool=True,
+                         enable_void_element: bool=True,
+                         enable_optional_tag: bool=True,
+                         enable_type_attribute: bool=True,
+                         enable_concerns_separation: bool=True,
+                         enable_protocol: bool=True,
+                         enable_names: bool=True,
+                         enable_capitalization: bool=True,
+                         enable_quotation: bool=True,
+                         enable_indentation: bool=True,
+                         enable_formatting: bool=True,
+                         enable_boolean_attribute: bool=True,
+                         enable_invalid_attribute: bool=True,
+                         enable_void_zero: bool=True,
+                         enable_invalid_handler: bool=True,
+                         enable_http_equiv: bool=True,
+                         enable_extra_whitespace: bool=True,
                          htmllint_ignore: typed_list(str)=()):
         """
-        :param htmllint_ignore: List of checkers to ignore.
+        :param enable_doctype:
+            Enable or disable doctype checker.
```

Comment:
maybe you should add an example here, it is not quite clear what this setting does